seaborn/utils.py

```diff
@@ -380,13 +402,36 @@ def load_dataset(name, **kws):
         Name of the dataset (`name`.csv on
         https://github.com/mwaskom/seaborn-data).  You can obtain list of
         available datasets using :func:`get_dataset_names`
+    cache : boolean, optional
+        If True, then cache data locally and use the cache on subsequent calls
+    data_home : string, optional
+        The directory in which to cache data. By default, uses ~/seaborn_data/
```

Comment:
Oops I missed the environment variable route (well I saw it, I just didn't have it in mind) that's a good solution.